lib/web/fetch/body.js

```diff
@@ -424,16 +418,18 @@ function mixinBody (prototype) {
  * @param {(value: unknown) => unknown} convertBytesToJSValue
  * @param {Response|Request} instance
  */
-async function consumeBody (object, convertBytesToJSValue, instance) {
+function consumeBody (object, convertBytesToJSValue, instance) {
   webidl.brandCheck(object, instance)
 
   // 1. If object is unusable, then return a promise rejected
   //    with a TypeError.
   if (bodyUnusable(object)) {
-    throw new TypeError('Body is unusable: Body has already been read')
+    return Promise.reject(new TypeError('Body is unusable: Body has already been read'))
   }
 
-  throwIfAborted(object[kState])
+  if (object[kState].aborted) {
+    return Promise.reject(new DOMException('The operation was aborted.', 'AbortError'))
+  }
```

Comment:
Actually I thin aborted is a subtype of the object being unusable. Does it have to be a DOMException?